src/main/java/seedu/address/logic/commands/ExportEventXmlCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.io.File;
+import java.util.List;
+
+import javax.xml.parsers.DocumentBuilder;
+import javax.xml.parsers.DocumentBuilderFactory;
+import javax.xml.transform.Transformer;
+import javax.xml.transform.TransformerFactory;
+import javax.xml.transform.dom.DOMSource;
+import javax.xml.transform.stream.StreamResult;
+
+import org.w3c.dom.Attr;
+import org.w3c.dom.Document;
+import org.w3c.dom.Element;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+
+/**
+ * Export an Event information as an XML
+ */
+public class ExportEventXmlCommand extends Command {
+
+    public static final String COMMAND_WORD = "exporteventxml";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": exports an XML file of the event "
+            + "given the file path to the xml file\n"
+            + "Parameters: INDEX (must be a positive integer)\n"
+            + "Example: " + COMMAND_WORD + " 1";
+
+    private static final String MESSAGE_EXPORT_EVENT_SUCCESS = "Event at %1$d exported"
```

Comment:
```suggestion
    private static final String MESSAGE_EXPORT_EVENT_SUCCESS = "Event at %1$d exported "
```
Improving grammar by adding a space between 'exported' and 'to your Desktop'.